ws-server/src/server.rs

```diff
@@ -168,3 +172,29 @@ async fn background_task(
 		}
 	}
 }
+
+/// Builder to configure and create a JSON-RPC Websocket server
+#[derive(Debug)]
+pub struct Builder {
```

Comment:
We should probably split this into a `Builder` and `Settings` to keep things sane.